manifests/install.pp

```diff
@@ -233,9 +233,12 @@
         }
       }
 
-      if "${::python::version}" =~ /^python3/ { #lint:ignore:only_variable_string
+      if "${python::version}" =~ /^python3/ and ($facts['os']['family'] == 'RedHat') and (versioncmp($facts['os']['release']['major'], '7') >= 0) { #lint:ignore:only_variable_string
```

Comment:
Does this not work with just:
`if $python::version`, which would fix the lint error and also avoid needing to ignore the lint error?